roles/kubespray-defaults/defaults/main.yaml

```diff
@@ -304,6 +308,8 @@ openstack_lbaas_monitor_timeout: "30s"
 openstack_lbaas_monitor_max_retries: "3"
 openstack_cacert: "{{ lookup('env','OS_CACERT') }}"
 
+cloud_config_file: "{{ kube_config_dir }}/cloud_config"
```

Comment:
Is this related to kubeadm etcd module?

Reply:
nope it's a bad rebase. I'll remove this